tests/bpftrace.cpp

```diff
@@ -604,15 +553,15 @@ TEST(bpftrace, add_probes_uprobe_string_offset)
 TEST(bpftrace, add_probes_uprobe_cpp_symbol)
 {
   for (const std::string provider : { "uprobe", "uretprobe" }) {
-    std::string prog = provider + ":/bin/sh:cpp:cpp_mangled{}";
-    ast::Probe *probe = parse_probe(prog);
-
     auto bpftrace = get_strict_mock_bpftrace();
+    bpftrace->feature_ = std::make_unique<MockBPFfeature>(false);
```

Comment:
Why disable features in this test? (also done in a few others below)

Reply:
This is to disable uprobe_multi and check that full expansion happens in such a case.